sys/riscv/riscv/trap.c

```diff
@@ -582,13 +591,6 @@ do_trap_user(struct trapframe *frame)
 		    (uintcap_t)frame->tf_stval, exception, 0);
```

Comment:
Note here that RISC-V has kept SEGV_LOADTAG handling here in the separate clause.